bin/wait_then_run

```diff
@@ -32,4 +32,5 @@ else
   python3 hmt_escrow/job.py
   python3 hmt_escrow/storage.py
   python3 hmt_escrow/eth_bridge.py
+  python3 hmt_escrow/tests/test_ipns.py
```

Comment:
Can this be moved to: https://github.com/hCaptcha/hmt-escrow/blob/master/bin/test?